assembly-multiuser/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/multiuser.properties

```diff
@@ -0,0 +1,84 @@
+#
```

Comment:
the path of this file should be within "WEB-INF/classes/che", not codenvy

Reply:
done